srv/runner/runner.py

```diff
@@ -3,13 +3,14 @@
 import logging
 import subprocess
 import time
+import click
```

Comment:
click is a third-party package, so it has to be imported in a separate import block (https://www.python.org/dev/peps/pep-0008/#id23)